jenkins/swift-functional-tests/30-run-tests.sh

```diff
@@ -10,5 +10,10 @@ cd $WORKSPACE
 mkdir jenkins-logs
 echo "Creating jenkins-log directory."
 cp -R /opt/stack/logs/* jenkins-logs/
+if [[ -f "/var/log/messages" ]]; then
+    sudo cp /var/log/messages jenkins-logs/messages
+elif [[ -f "/var/log/syslog" ]]; then
```

Comment:
Don't use elif here. It makes sense to have both syslog and messages when the 2 are available